deerlab/dipolarkernel.py

```diff
@@ -60,9 +61,15 @@ def dipolarkernel(t, r, *, pathways=None, mod=None, bg=None, method='fresnel', e
         
         The default is ``'fresnel'``.
 
+    orisel : array_like, optional 
+        Probability distribution of possible orientations of the interspin vector to account for orientation selection. Must be 
+        vector defined for uniformly distributed orientations in the range [0,π/2]. If speficied as ``None`` (by default), a uniform distribution is assumed. 
+        Requires the ``'grid'`` method.
+
     excbandwidth : scalar, optional
         Excitation bandwidth of the pulses in MHz to account for limited excitation bandwidth [4]_.
-    
+        Requires the ``'grid'`` or ``'integral'`` methods.
```

Comment:
It's not implemented for the `integral` method. Please implement, good to have a second decoupled implementation.

Reply:
Done